src/app/about/page.js

```diff
@@ -1,30 +1,30 @@
 import Image from "next/image"
 import mockup from "../images/mockup.png"
-import name from "../images/logo.png"
+import name from "../images/demo.png"
```

Comment:
Could you rename the reference for the import from "name"  to "demo", and update the corresponding references? Thank you!

Reply:
I renamed the reference and updated the corresponding references.